sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosFullTextPath.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.azure.cosmos.implementation.apachecommons.lang.StringUtils;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Path settings within {@link CosmosFullTextPolicy}
+ */
+public class CosmosFullTextPath {
+    @JsonProperty(Constants.Properties.PATH)
+    private String path;
+    @JsonProperty(Constants.Properties.LANGUAGE)
+    private String language;
+    private JsonSerializable jsonSerializable;
+
+    /**
+     * Constructor
+     */
+    public CosmosFullTextPath() { this.jsonSerializable = new JsonSerializable(); }
+
+    /**
+     * Gets the path for the cosmosFullText.
+     *
+     * @return path
+     */
+    public String getPath() {
+        return path;
+    }
+
+    /**
+     * Sets the path for the cosmosFullText.
+     *
+     * @param path the path for the cosmosFullText.
+     * @return CosmosFullTextPath
+     */
+    public CosmosFullTextPath setPath(String path) {
+        if (StringUtils.isEmpty(path)) {
+            throw new NullPointerException("full text search path is either null or empty");
+        }
+
+        if (path.charAt(0) != '/' || path.lastIndexOf('/') != 0) {
```

Comment:
So we don't allow nested paths to have a language other than the default?  Is that something worth validating in the client?

Reply:
At-least in .NET nested paths are allowed. 
Again IMO we can leave client implementation some what flexible and delegate such to the service (might help on future proofing as well)

Reply:
Fixed this, nested paths are allowed.